.github/workflows/build.yml

```diff
@@ -61,17 +76,138 @@ jobs:
+              exit 1
+              ;;
+          esac
+          source ./set-ubuntu-${{ matrix.ubuntuName }}.sh
+          source ./set-raven-platform-${arch}.sh
+          ./build-deb.sh
+          echo "DEB_ARCH=$arch" >> $GITHUB_ENV
+          echo "UBUNTU_VERSION=$DISTRO_VERSION" >> $GITHUB_ENV
+        working-directory: ravendb/scripts/linux/pkg/deb
+        env:
+          RAVENDB_VERSION: ${{ inputs.version }}
+
+      - name: Upload deb package to artficats
+        uses: actions/upload-artifact@v3
+        with:
+          name: package-${{ matrix.ubuntuName }}-${{ matrix.arch }}
+          path: ravendb/scripts/linux/pkg/deb/dist/${{ env.UBUNTU_VERSION }}/ravendb*.deb
+
+      - name: Upload deb package to s3
+        run: |
+          set -e
+          set -x
+          #export DRY_RUN=1
+
+          PACKAGES_DIST_DIR=ravendb/scripts/linux/pkg/deb/dist
+
+          ls -R $PACKAGES_DIST_DIR
+
+          if [ "${{ env.IS_NIGHTLY }}" == "true" ]; then
+
+            NIGHTLY_VERSION_DATE_PART=$(date "+%Y%M%d-%H%m" -u)
+            export BUILD_VERSION="$NIGHTLY_VERSION_DATE_PART-Nightly"
+          elif [ "${{ env.IS_STABLE }}" == "true" ]; then
+            export BUILD_VERSION="${{ inputs.triggering_build_number }}"
+          else
+            export BUILD_VERSION="${{ inputs.triggering_build_number }}-${{ env.TRIGGERING_BUILD_TYPE }}"
+          fi
+
+          export COMMENT="RavenDB_${RAVENDB_VERSION}_DEB"
+          export S3UPLOADER="./current/s3uploader.sh"
+
+          "$S3UPLOADER" --category "RavenDB for Ubuntu ${{ env.UBUNTU_VERSION }} ${{ matrix.arch }} DEB"   --filename "${PACKAGES_DIST_DIR}/${{ env.UBUNTU_VERSION }}/ravendb_${RAVENDB_VERSION}-0_ubuntu.${{ env.UBUNTU_VERSION }}_${{ env.DEB_ARCH }}.deb"
+        env:
+          RAVENDB_VERSION: ${{ inputs.version }}
+          BUCKET_NAME: ${{ secrets.BUCKET_NAME }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+          SECRET_PASS: ${{ secrets.SECRET_PASS }}
+
+  linux:
+    strategy:
+      matrix:
+        arch: [ x64, arm32v7, arm64v8 ]
+      fail-fast: false
+    runs-on: ubuntu-latest
+    needs: debPackage
+    steps:
+      - name: Get ravendb majorminor version & set repo
+        run: |
+          v=$(echo ${{ inputs.version }} | cut -d. -f1,2)
+          echo "RAVEN_MAJOR_MINOR=$v" >> $GITHUB_ENV
+          [[ ${{ env.IS_NIGHTLY }} == true ]] && DOCKER_REPO="$DOCKER_REPO-nightly"
+          echo "DOCKER_REPO=$DOCKER_REPO" >> $GITHUB_ENV
+
+      - uses: actions/checkout@v3
+        with:
+          repository: 'hiddenshadow21/ravendb'
+          ref: "v${{ env.RAVEN_MAJOR_MINOR }}"
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: package-focal-${{ matrix.arch }}
+          path: downloads
+
+      - name: Move artifacts to correct folder

```

Comment:
Doesn't docker build script take files from the `artifacts` dir by default?

Reply:
Only `tar` files, `.deb` by default are in `docker/ravendb-ubuntu/` dir.